### PR TITLE
fixes add another button size

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|upload|screen shot 2015-11-19 at 10 04 10|

### CODE REVIEW

File: djangocms_admin_style/sass/components/_icons.scss [hunk 1]
```diff
@@ -177,6 +177,8 @@ a[id*="clocklink"] {
 .delete-related {
     @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
     display: inline-block;
+    width: 15px;
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_icons.scss]: please add the comment in code :)

File: djangocms_admin_style/sass/components/_icons.scss [hunk 2]
```diff
@@ -177,6 +177,9 @@ a[id*="clocklink"] {
 .delete-related {
     @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
     display: inline-block;
+    /* make sure that add another button has correct width and height on advanced setting */
```
Comment (FinalAngel) [inline djangocms_admin_style/sass/components/_icons.scss]: ``// make sure that add another button has correct width and height on advanced setting #251``

**Comment by FinalAngel** [comment]
LGTM

**Comment by vxsx** [comment]
:+1:

**Pull request merged**